source/common/common/utility.h

```diff
@@ -687,12 +688,15 @@ template <class Value> class BigTrieEntry {
 template <class Value> class SmallTrieEntry {
 public:
   Value value_{};
-  SmallTrieEntry* operator[](uint8_t i) const {
+  const SmallTrieEntry* operator[](uint8_t i) const {
```

Comment:
If this is never used, what happens if you comment it out?

Reply:
Oh yeah, of course it's used because the find() is const, I'm totally wrong.